default/universe_tables.txt

```diff
@@ -46,6 +46,7 @@ SPIRAL_3                        0           0           0       0       0
 SPIRAL_4                        0           0           0       0       0       0       0               0
 CLUSTER                         0           0           0       0       0       0       0               0
 ELLIPTICAL                      0           0           0       0       0       0       0               0
+DISC                            0           0           0       0       0       0       0               0
```

Comment:
Thanks a lot. I knew it had to be something simple and obvious. Facepalm moment of the day... :wink: